.werft/vm/vm.ts

```diff
@@ -90,14 +90,15 @@ export function waitForVM(options: { name: string, timeoutSeconds: number, slice
  * Copies the k3s kubeconfig out of the VM and places it at `path`
  * If it doesn't manage to do so before the timeout it will throw an Error
  */
-export function copyk3sKubeconfig(options: { path: string, timeoutMS: number, slice: string }) {
+export function copyk3sKubeconfig(options: { name: string, path: string, timeoutMS: number, slice: string }) {
     const werft = getGlobalWerftInstance()
     const startTime = Date.now()
     while (true) {
 
         const status = exec(`ssh -i /workspace/.ssh/id_rsa_harvester_vm ubuntu@127.0.0.1 -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no 'sudo cat /etc/rancher/k3s/k3s.yaml' > ${options.path}`, { silent: true, dontCheckRc: true, slice: options.slice })
 
         if (status.code == 0) {
+            exec(`kubectl --kubeconfig ${options.path} config set clusters.default.server https://${options.name}.kube.gitpod-dev.com:6443`, { silent: true, dontCheckRc: true, slice: options.slice });
```

Comment:
I don't think we want `dontCheckRc: true` in this case? If the kubectl command fails won't the k3s config be invalid?

Reply:
I removed the part.